packages/webdriver-utils/src/providers/genericProvider.js

```diff
@@ -33,6 +33,12 @@ export default class GenericProvider {
     this.debugUrl = null;
     this.header = 0;
     this.footer = 0;
+    this.statusBarHeight = 0;
+    this.pageXShiftFactor = 0;
+    this.pageYShiftFactor = 0;
+    this.currentTag = null;
+    this.removeElementShiftFactor = 50000;
```

Comment:
What's the use of `removeElementShiftFactor` ?

Reply:
This is to make the coordinates incorrect when the element is not visible on screen. So when scroll values don't match the top and bottom becomes negative causing it to be 0 and hence not drawing it.